pkg/nodeclient/itzoclient.go

```diff
@@ -446,10 +431,6 @@ func StreamLogsEndpoint(unitName string, withMetadata bool) string {
 	return fmt.Sprintf("rest/v1/logs/%s?follow=1&metadata=%d", unitName, metadataflag)
 }
 
-func PortForwardEndpoint() string {
-	return fmt.Sprintf("rest/v1/portforward/")
-}
-
```

Comment:
IIRC VK does not implement the port-forward endpoint, but we have implemented it in itzo. I'd keep it in here even though it's unused, it could a good and straightforward change in VK to add it to their API.

Reply:
:+1: reverted